src/OpenTelemetry/README.md

```diff
@@ -313,14 +313,17 @@ the following content:
 }
 ```
 
+Note: There is no need to restart the process to start this feature.
+
```

Comment:
nit: remove the extra white line here.

I wonder if this sounds better: "The self-diagnostics feature can be enabled/disabled while the process is running. To enable self-diagnostics, go to the ...". In this way we avoid have both "Note" and "Tip" (which might look weird).